training/06-inner-blocks.md

```diff
@@ -124,29 +124,10 @@ First, look at the [`package.json`](https://github.com/10up/gutenberg-lessons/tr
     }
 ```
 
-Second, we need to register the block in [`blocks.php`](https://github.com/10up/gutenberg-lessons/tree/trunk/themes/tenup-theme/includes/blocks.php). There are two steps to this. Step one: find the comment `// Require custom blocks` and the line `require_once TENUP_THEME_BLOCK_DIR . '/inner-blocks-one-complete/register.php';`. We want to duplicate this line and reference the starter register file:
```

Comment:
It seems like we don't need to register those blocks manually anymore since [they are being automatically added automatically](https://github.com/10up/gutenberg-lessons/blob/trunk/themes/tenup-theme/includes/blocks.php#L50).